lib/geo-web-content/mediaGallery.ts

```diff
@@ -0,0 +1,74 @@
+import { useState, useEffect } from "react";
+import { AssetId, AccountId } from "caip";
+import BN from "bn.js";
+import { CeramicClient } from "@ceramicnetwork/http-client";
+import { GeoWebContent } from "@geo-web/content";
+import type { Encoding } from "@geo-web/types";
+import type { CID } from "multiformats/cid";
+import { NETWORK_ID } from "../constants";
+
+export interface MediaGalleryItem {
```

Comment:
Does `MediaObject` from `@geo-web/types` not work here?

Reply:
It does and the keys are the same so there was no reason for another one.